integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -1008,6 +1062,46 @@ private static void verifyMemberHealth(String adminServerPodName, List<String> m
         });
   }
 
+  private void verifyWebLogicVersioninPod(int replicaCount) {
```

Comment:
Change the method name to verifyIntrospectVersionLabelInPod()  to represent the action.

Reply:
Do we set the IntrospectVersion explicitly on domain resource using  spec.introspectVersion syntax  while creating the domain ?

Reply:
changed

Reply:
yes